setup.py

```diff
@@ -0,0 +1,16 @@
+import os
+from setuptools import setup, find_packages
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+
```

Comment:
This works here, but will not work in general so wouldn't recommend this as a practice.